packages/analytics-js-integrations/src/integrations/FacebookPixel/browser.js

```diff
@@ -48,6 +48,7 @@ class FacebookPixel {
     this.userIdAsPixelId = config.userIdAsPixelId || [];
     this.whitelistPiiProperties = config.whitelistPiiProperties;
     this.useUpdatedMapping = config.useUpdatedMapping;
+    this.skipInitialization = config.skipInitialization;
```

Comment:
We're still initialising the destination. I recommend we call it `loadIntegration` (defaults to true) to also match with the SDK's `load` API option.

https://www.rudderstack.com/docs/sources/event-streams/sdks/rudderstack-javascript-sdk/load-js-sdk/#integrationopts:~:text=the%20data%20plane.-,loadIntegration,-Boolean

Moreover, the `integrations` object specified in the `load` API also supports `loadIntegration` option per destination.
For example,
https://github.com/rudderlabs/rudder-sdk-js/blob/3e8293a0d4526c293355e204e6e49273f4244429/packages/analytics-js-integrations/src/integrations/Posthog/browser.js#L44
CC @krishna2020